src/activities/binary_bulb/Binary_bulb.qml

```diff
@@ -0,0 +1,369 @@
+/* GCompris - binary_bulb.qml
```

Comment:
the filename should be BinaryBulb.qml

Reply:
These files were created by running that shell script. Is it that important to change the name of the file?

Reply:
For this case, the issue is not really important but we need to have the same coding rules everywhere, so yes.
If everyone codes like he wants, the project will be difficult to read everytime we change files to read written by another author.

Reply:
I have tried to use the repeater. Hope you pass it this time :)